nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py

```diff
@@ -1,8 +1,17 @@
 """Authors: Cody Baker and Ben Dichter."""
-from spikeextractors import SpikeGLXRecordingExtractor
+from spikeextractors import SpikeGLXRecordingExtractor, SubRecordingExtractor
 
 from ..baserecordingextractorinterface import BaseRecordingExtractorInterface
 
 
 class SpikeGLXRecordingInterface(BaseRecordingExtractorInterface):
     RX = SpikeGLXRecordingExtractor
+
+    def convert_data(self, nwbfile, metadata_dict: None, stub_test: bool = False, sync_with_ttl: bool = True,
+                     **conversion_options):
```

Comment:
Do we need to have the **conversion_options here too? Makes sense in the abstract class but I don't know if it's needed here, unless it is getting passed to another function